root/edit/details/AddRemoveAlias.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * @flow
+ * Copyright (C) 2019 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+import React from 'react';
+
+import bracketed from '../../static/scripts/common/utility/bracketed';
+import formatEntityTypeName from '../../static/scripts/common/utility/formatEntityTypeName';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import isDateEmpty from '../../static/scripts/common/utility/isDateEmpty';
+import DescriptiveLink from '../../static/scripts/common/components/DescriptiveLink';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import locales from '../../static/scripts/common/constants/locales';
+import isolateText from '../../static/scripts/common/utility/isolateText';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AliasEntityTypeT = $ElementType<AliasEntityTypeT, 'entityType'>;
+
+type AddRemoveAliasEditT = {
+  display_data: {
+    alias: string,
+    begin_date: PartialDateT,
+    end_date: PartialDateT,
+    ended: boolean,
```

Comment:
But we could still use the chance to start storing it in `Edit::Alias::Delete` while we're doing this, so that we have better data if we ever want to make these undoable. Opinion?

Reply:
Ended is a boolean so should it be conditionally stored ?

Reply:
In the usecase I've switched it to use ended = false incase there are where ended is null. Let me know if there is something wrong with it

Reply:
Opened a ticket for storing ended in the edit data, as @reosarevok suggested above, and pushed a commit for it.